dialog/file.go

```diff
@@ -311,8 +318,12 @@ func (f *fileDialog) setSelected(file *fileDialogItem) {
 	f.selected = file
 
 	if file == nil || file.location.String()[len(file.location.Scheme())+3:] == "" {
-		f.fileName.SetText("")
-		f.open.Disable()
+		// keep user input while navigating
+		// in a FileSave dialog
+		if !f.file.save {
```

Comment:
When I tap a file in save mode it should fill this with that file name...
Doesn't this code stop that functionality?

Reply:
No this will only execute if file == nil, eg when nothing is selected. If you tap a file the filename will be entered. If you navigate around (only clicking folders) the last tapped filename will remain in the entry.